pfcpiface/parse-pdr.go

```diff
@@ -212,7 +227,7 @@ func (p *pdr) parsePDI(pdiIEs []*ie.IE, appPFDs map[string]appPFD) error {
 	return nil
 }
 
-func (p *pdr) parsePDR(ie1 *ie.IE, seid uint64, appPFDs map[string]appPFD) error {
+func (p *pdr) parsePDR(ie1 *ie.IE, seid uint64, appPFDs map[string]appPFD, upf *upf) error {
```

Comment:
Do we need to pass entire UPF object or just ip_pool struct is enough?

Reply:
its just pointer passing. might need other values later. So keeping as it is.

Reply:
I was thinking in terms layer/scope instead pointer or not.